pkg/runtime/runtime_test.go

```diff
@@ -2149,3 +2149,31 @@ func testSecurity(t *testing.T) security.Handler {
 
 	return sec
 }
+
+func testGetOtelServiceName(t *testing.T) {
```

Comment:
This test isn't currently being executed.

```suggestion
 TestGetOtelServiceName(t *testing.T) {
```

Reply:
Fair enough - I'm quite new to Go and was just executing the test by itself via the CLI.